enos/cli.py

```diff
@@ -76,6 +76,8 @@ def up(**kwargs):
                          deployment [default: ./reservation.yaml].
     -h --help            Show this help message.
     --force-deploy       Force deployment [default: False].
+    --pull               Preinstall software (e.g pull docker images)
```

Comment:
I would say "Only preinstall ...", because with the actual sentence, it seems that if you don't activate that option, then images will not be pulled.